cfd_protocol/src/lib.rs

```diff
@@ -13,6 +13,7 @@ use bdk::{
     descriptor::Descriptor,
     miniscript::{descriptor::Wsh, DescriptorTrait},
 };
+use bitcoin::PrivateKey;
```

Comment:
Not sure if better but maybe the protocol should work with `secp256k1::SecretKey`s instead of the `bitcoin` ones?

Reply:
Good catch. This just came in because we copied over something from `thor` (which didn't need this type anyway :stuck_out_tongue:).